src/docs/metrics/social-dominance/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Social Dominance
+author: Ivan
+date: 2020-04-13
+# REF metrics-hub/metricshub/social_dominance.py
+---
+
+## Definition
+
+Social Dominance is build on top of the [Social Data](/metrics/details/social-data).
+
+Social Dominance for an asset compares the [Social
+Volume](/metrics/social-volume) of that asset to the combined social volume of
+all available assets.
+
+Social Dominance for arbitrary search term compares the [Social
+Volume](/metrics/social-volume) of that search term to the total number of
+documents available. This essentially counts how many percent of all documents
+match this search term.
+
+Social Dominance of 50 for an asset means that half of all messages/posts regarding
+assets are discussing exactly this asset.
+
+## Access
+
+[Restricted Access](/metrics/details/access#restricted-access).
+
+---
+
+## Measuring Unit
+
+% between 0 and 100.
+
+---
+
+## Data Type
+
+[Timeseries Data](/metrics/details/data-type#timeseries-data)
+
+---
+
+## Frequency
+
+[Five-Minute Intervals](/metrics/details/frequency#five-minute-frequency)
+
+---
+
+## Latency
+
+[Social Data Latency](/metrics/details/latency#social-data-latency)
+
+---
+
+## Available Assets
+
+Available for [these
+assets](<https://api.santiment.net/graphiql?variables=&query=%7B%0A%20%20getMetric(metric%3A%20%22social_dominance_total%22)%20%7B%0A%20%20%20%20metadata%20%7B%0A%20%20%20%20%20%20availableSlugs%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A>)
```

Comment:
why link in "<" ?

Reply:
Not sure, my autoformatter does this with longer links